templates/16-04/azure.html

```diff
@@ -0,0 +1,145 @@
+      </p>
+    </div>
+    <div class="col-5 u-align--center u-vertically-center u-hide--small">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
+        alt="",
+        width="350",
+        height="100",
+        hi_def=True,
+        loading="auto",
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Upgrading Ubuntu 16.04 LTS on Azure to Ubuntu Pro
+      </h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3 class="p-muted-heading">
+        New or redeployed instances
+      </h3>
+      <p>
+        For fresh Ubuntu 16.04 instances, or if you are able to redeploy your workload, we recommend using Ubuntu Pro images. <a href="/azure/pro">Ubuntu Pro for Azure</a> is a premium image, optimised for Azure with security and compliance features built in, and optimised for the public cloud.
+      </p>
+      <p>
+        Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. It is ideal for companies that have embraced open source for their new products.
+      </p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial?tab=Overview">Launch Ubuntu Pro 16.04 LTS on Azure</a>
+      </p>
+    </div>
+    <div class="col-6">
+      <h3 class="p-muted-heading">
+        Upgrading existing instances
+      </h3>
+      <p>
+        If you are looking to upgrade your existing Ubuntu 16.04 LTS Azure instances you can <a class="p-link--external" href="https://azuremarketplace.microsoft.com/uk-ua/marketplace/apps/Canonical.UbuntuAdvantageSupport?tab=Overview">enable the Ubuntu Advantage subscription directly from the Azure marketplace</a>.
+      </p>
+      <p>
+        Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Migrate Azure Ubuntu instances to a later LTS release
+      </h2>
+      <p>
+        You should <a class="p-link--external" href="https://info.microsoft.com/ww-ondemand-expert-tips-to-easily-migrate-your-ubuntu-stack-to-azure.html">evaluate if you can easily move your instances</a> to either Ubuntu 20.04 LTS or Ubuntu 18.04 LTS before April 30, 2021. In-place upgrades of cloud images from LTS to LTS are supported by Canonical.
+      </p>
+      <p>
+        <a class="js-invoke-modal" shref="/azure/contact-us">
+          Contact us with your migration questions&nbsp;&rsaquo;
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
```

Comment:
Lech replied in the copy doc about the image so not sure if you want to add it here?

Reply:
Done, I think it is stupid, but done.